base/strings/util.jl

```diff
@@ -446,7 +448,7 @@ end
 # check for pure ASCII-ness
 
 function ascii(s::String)
-    for (i, b) in enumerate(s.data)
+    for (i, b) in enumerate(bytes(s))
```

Comment:
What is `bytes`?  I thought we were now using `convert(Vector{UInt8}, s)` or maybe `Vector(s)` or `Vector{UInt8}(s)`?